diff --git a/theano/gof/cmodule.py b/theano/gof/cmodule.py
index edd41f6..87fa399 100644
--- a/theano/gof/cmodule.py
+++ b/theano/gof/cmodule.py
@@ -20,7 +20,6 @@ import platform
 import distutils.sysconfig
 import warnings
 
-import numpy.distutils
 
 import theano
 from theano.compat import PY3, decode, decode_iter
@@ -1621,7 +1620,7 @@ def get_gcc_shared_library_arg():
 
 
 def std_include_dirs():
-    numpy_inc_dirs = numpy.distutils.misc_util.get_numpy_include_dirs()
+    numpy_inc_dirs = [numpy.get_include()]
     py_inc = distutils.sysconfig.get_python_inc()
     py_plat_spec_inc = distutils.sysconfig.get_python_inc(plat_specific=True)
     python_inc_dirs = ([py_inc] if py_inc == py_plat_spec_inc
diff --git a/theano/tensor/blas.py b/theano/tensor/blas.py
index ace7faa..b2e71c7 100644
--- a/theano/tensor/blas.py
+++ b/theano/tensor/blas.py
@@ -131,7 +131,6 @@ import os
 import time
 
 import numpy as np
-import numpy.distutils
 try:
     import numpy.distutils.__config__  # noqa
 except ImportError:
